Class/practice4.1/PizzaBuilder.cpp

```diff
@@ -27,6 +27,8 @@ void PizzaBuilder::addIngredient(const std::string& ingredient) {
     }
 }
 
-Pizza PizzaBuilder::getPizza() const {
-    return pizza;
+[[nodiscard]] Pizza PizzaBuilder::getPizza() {
+    Pizza tempPizza = pizza; // Create a copy of the pizza
```

Comment:
Or we can even do 
```
Pizza tempPizza= std::move(pizza);
return tempPizza;
```
that will implicitly call default constructor for pizza object and we will save copy constructor call in this case